ci/jscrambler-code-integrity.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+# This is a basic workflow to help you get started with Using Jscrambler Code Integrity Action,
```

Comment:
Can you add what does this worklow/action do, how it does that and what all is required for it to run?

Reply:
Sure